### DISCUSSION TITLE
Exogenous variables in Hierarchical forecasting

### DISCUSSION THREAD

**Discussion by Atumbo** [alan-turing-institute/sktime#7328] [Q&A]
Is it possible to add exogenous variables to hierarchical forecasting in sktime? So that it is aggregated in the right way to different levels? Or how to add them?

**Comment by felipeangelimvieira** [comment]
It is possible. When I forecast with hierarchical timeseries the `X` dataframe has the same index structure of `y`. I believe the aggregation of exogenous variables really depend on your usecase - not always the exogenous variable for the total equals the sum of the bottom levels. However, if this is your case, I think you can easily use sktime's Aggregator to do it.

**Reply by Atumbo** [reply]
Yes, I noticed that summing does not always make sense. 
Is there a way to implement other strategies?
